Sources/PointFree/__Generated__/DerivedPartialIsos.swift

```diff
@@ -0,0 +1,12 @@
+// Generated using Sourcery 0.15.0 — https://github.com/krzysztofzablocki/Sourcery
```

Comment:
oh ha forgot to delete this (and the corresponding makefile commands) after everything was moved to PointFreeRouter.